index.html

```diff
@@ -74,16 +74,22 @@ <h5 class="quote-author" id="quoteAuthorTag">- Steve Jobs</h5>
         </div>
         <!-- Share Button -->
         <div class="share-buttons">
-          <a href="#" class="whatsapp-btn">
-            <i class="fa-brands fa-square-whatsapp fa-2xl" style="color: #00ff33;"></i>
-          </a>
+          <div class="washare">
+            <a href="#" class="whatsapp-btn">
+              <i class="fa-brands fa-square-whatsapp fa-2xl" style="color: #00ff33;"></i>
+            </a>
+          </div>
           <div class="fbshare">
             <i class="fa-brands fa-facebook fa-2xl" style="color: #0084ff;"></i>
```

Comment:
no need to fix whatapp share button.
dont do anything unless you are assign to do.
you should have read the contribution rules.